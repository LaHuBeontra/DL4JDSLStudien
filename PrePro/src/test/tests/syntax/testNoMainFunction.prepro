import vec3 p1;

vec3 res = p1 * p1;

export p1, res;
