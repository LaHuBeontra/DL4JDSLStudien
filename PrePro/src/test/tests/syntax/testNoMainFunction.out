DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='p1', variableShape=[4, 3], startColumn=0, length=3}, DataSetEntry{variableName='res', variableShape=[4, 3], startColumn=3, length=3}],
dataSet=
[[42.00,  0.00,  0.00,  1,764.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00,  1.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00,  25.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00,  36.00,  0.00,  0.00]]

constants={}

calculated_variables=
p1=[[42.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00]]
res=[[1,764.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00],  
 [25.00,  0.00,  0.00],  
 [36.00,  0.00,  0.00]]
}